test/functional/uvm_vsmb_test.go

```diff
@@ -55,7 +57,8 @@ func TestVSMB_Writable(t *testing.T) {
 
 	opts := uvm.NewDefaultOptionsWCOW(t.Name(), "")
 	opts.NoWritableFileShares = true
-	vm, _, _ := tuvm.CreateWCOWUVMFromOptsWithImage(context.Background(), t, opts, "microsoft/nanoserver")
+	vm, _, uvmScratchDir := tuvm.CreateWCOWUVMFromOptsWithImage(context.Background(), t, opts, "microsoft/nanoserver")
+	defer os.RemoveAll(uvmScratchDir)
```

Comment:
This shouldn't be necessary since `uvmScratchDir` is created with `t.TempDir()`. 
Are you seeing it not be cleaned up?

Reply:
Yup. I noticed. Still hacking away at it for now.

Reply:
kk, ill stop nit-picking then

Reply:
Ohh. Feel free to nit-pick! I do not mind at all.